backend/YouTubeV2.Api.Tests/VideoControllerTests/GetVideoMetadataAsyncTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System.Security.Claims;
+using YouTubeV2.Application.Model;
+using YouTubeV2.Application.Services;
+using YouTubeV2.Application;
+using Microsoft.AspNetCore.Identity;
+using YouTubeV2.Api.Enums;
+using Newtonsoft.Json;
+using System.Net.Http;
+using System.Net;
+using System.Text;
+using YouTubeV2.Application.DTO;
+using System.Web;
+using FluentAssertions;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+
+namespace YouTubeV2.Api.Tests.VideoControllerTests
+{
+    [TestClass]
+    public class GetVideoMetadataAsyncTests
+    {
+        private WebApplicationFactory<Program> _webApplicationFactory = null!;
+        private readonly Mock<IUserService> _userService = new();
+
+        [TestInitialize]
+        public async Task Initialize()
+        {
+            _userService
```

Comment:
do we need it?

Reply:
I think that now we need it after i added validation